docs/src/index.md

```diff
@@ -1,91 +1,75 @@
 # KernelAbstractions
 
-`KernelAbstractions.jl` is a package that allows you to write GPU-like kernels that
-target different execution backends. It is intended to be a minimal, and performant
-library that explores ways to best write heterogenous code.
+[`KernelAbstractions.jl`](https://github.com/JuliaGPU/KernelAbstractions.jl) (KA) is
+a package that allows you to write GPU-like kernels targetting different
+execution backends. KA intends to be a minimal and
+performant
+library that explores ways to write heterogeneous code. Although parts of
+the package are still experimental, it has been used successfully as part of the
+Exascale Computing Project to run Julia code on pre-Frontier and pre-Aurora
+systems. Currently missing from the documentation is
+ * how to debug kernels, and
+ * how to profile kernels.
 
 !!! note
-    While `KernelAbstraction.jl` is focused on performance portability, it is GPU-biased
-    and therefore the kernel language has several constructs that are necessary for good
-    performance on the GPU, but may hurt performance on the CPU.
-
-## Quickstart
-
-### Writing your first kernel
-
-Kernel functions have to be marked with the [`@kernel`](@ref). Inside the `@kernel` macro
-you can use the [kernel language](@ref api_kernel_language). As an example the `mul2` kernel
-below will multiply each element of the array `A` by `2`. It uses the [`@index`](@ref) macro
-to obtain the global linear index of the current workitem.
-
+    While `KernelAbstraction.jl` focuses on performance portability, it
+    currently remains GPU-biased.
+    Therefore, the kernel language requires several specific constructs for good GPU
+    performance but may hurt CPU performance.
```

Comment:
Could you provide an example to back this up? The only thing I can think of is shmem usage, but I am sure there is more?

Reply:
There I'm out of luck. We need @vchuravy 's wisdom here.